modules/network-observability-operator-uninstall.adoc

```diff
@@ -9,9 +9,18 @@ You can uninstall the Network Observability Operator using the {product-title} w
 
 .Procedure
 
-. Uninstall the `FlowCollector` custom resource.
+. Remove the `FlowCollector` custom resource.
 .. Click *Flow Collector*, which is next to the *Network Observability Operator* in the *Provided APIs* column. 
 .. Click the options menu {kebab} for the *cluster* and select *Delete FlowCollector*.
 . Uninstall the Network Observability Operator.
 .. Navigate back to the *Operators* -> *Installed Operators* area. 
-.. Click the options menu {kebab} next to the  *Network Observability Operator* and select *Uninstall Operator*.
\ No newline at end of file
+.. Click the options menu {kebab} next to the  *Network Observability Operator* and select *Uninstall Operator*.
+. Remove the `FlowCollector` custom resource definition (CRD).
+.. Navigate to *Administration* -> *CustomResourceDefinitions*. 
+.. Look for *FlowCollector* and click the options menu {kebab}. 
+.. Select *Delete CustomResourceDefinition*.
++
+[IMPORTANT]
+====
+The Loki Operator and Kafka remain if they were installed and must be removed separately. Additionally, you might have remaining data stored in object store, and persistent volume that must be removed. 
```

Comment:
Requesting a revision on the note. Maybe something like... 

. Additionally, you might have the remaining data stored in an object store, and a persistent volume that must be removed.

Reply:
Agree, this is 100% missing articles THANK YOU! Adding a/an